pkg/skaffold/runner/build_deploy.go

```diff
@@ -122,9 +116,6 @@ func (r *Builder) Build(ctx context.Context, out io.Writer, artifacts []*latestV
 		return nil, err
 	}
 
-	// Update which images are logged.
-	r.AddTagsToPodSelector(bRes)
```

Comment:
can you point me to where this line moved to in the `Deployer`?

Reply:
addressed in https://github.com/GoogleContainerTools/skaffold/pull/5974